content/using/os/ship-troubleshooting.md

```diff
@@ -22,19 +22,19 @@ If your issue is not covered here, please check out our [Troubleshooting Knowled
 
 An ounce of prevention is worth a pound of cure, so let's first go over some best practices to keep your ship in working order.
 
-### Only boot with your keyfile once
+### Only boot your ship once
 
-Once your ship is booted with your keyfile, you should never use that same keyfile again. If you do boot with the same keyfile twice, any other ship on the network that your ship has communicated with will not be able to talk to your ship.
+Once your ship is booted, you should never boot it a second time. If you do boot it anew without special measures, you will experience trouble communicating with ships on the network you had talked to before.
```

Comment:
What's "booting"? Is starting it again after you've shut it down booting?

That's obviously not what's meant here, but I could see someone being confused by that.

Reply:
Hard agree. What we need is to fundamentally change what we call this operation. Boot is bad because everyone is confused by it. We have to stop using it. It may be the "correct" term but it brings too much baggage. For the same reason we call "functions" gates, we need to change this. @drbeefsupreme suggested last year we change it to "dawn" There was some technical objection to this but I think it works really well. That will have to be a secondary pr though as it will need to be consistently changed. 

Please revert this particular change.

Reply:
Well I mean the old copy reads like you need to cycle networking keys and get a new keyfile every time you wanna resume your ship

Reply:
I agree with @jalehman and @baudtack that saying "never boot a second time" is confusing to new people. But also I agree with @matildepark that it shouldn't sound like you need a new keyfile every time. So I don't think it should be reverted to the old commit, but should take into account both of these concerns. And we'll change the language to dawn soon.